mappings/net/minecraft/server/world/ChunkTicketManager.mapping

```diff
@@ -69,6 +69,7 @@ CLASS net/minecraft/class_3204 net/minecraft/server/world/ChunkTicketManager
 		ARG 4 argument
 	METHOD method_21623 getTicket (J)Ljava/lang/String;
 		ARG 1 pos
+	METHOD method_21683 toDumpString ()Ljava/lang/String;
```

Comment:
What's a dump string? How is different from `getDebugString`?

Reply:
a dumped string, as shown in debug command dump.